scripts/templates/component.tsx.hbs

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+className?: string;
+};
+
+const {{componentName}}: React.FC<Props> = ({ className, ...other }) => {
+    const classes = classNames(className, '{{className}}');
+
+    return
```

Comment:
The formatting here looks a little odd

Reply:
I can't seem to save it without formating. Tried with prettier ignore and disabling the extension but nothing